sdcm/sct_provision/user_data_objects/vector_dev.py

```diff
@@ -0,0 +1,32 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2025 ScyllaDB
+from dataclasses import dataclass
+
+from sdcm.provision.common.utils import configure_vector_target_script, install_vector_service
+from sdcm.sct_provision.user_data_objects import SctUserDataObject
+
+
+@dataclass
+class VectorDevUserDataObject(SctUserDataObject):
+
+    @property
+    def is_applicable(self) -> bool:
+        return self.params.get('logs_transport') == 'vector'
+
+    @property
+    def script_to_run(self) -> str:
+        script = install_vector_service()
+        host, port = self.test_config.get_logging_service_host_port()
+        script += configure_vector_target_script(
+            host=host, port=port, prometheus_url=self.test_config.prometheus_url)
```

Comment:
this looks race'y: monitor set must start configuring to execute `self.test_config.set_prometheus_url` before this.

Is there problem to configure monitoring to collect vector metrics from nodes by polling instead of pushing from vector?

Reply:
There's no such built-in option in vector 

Anyhow it's not racy, since we configure it first with the wrong address, and once monitoing  is ready we configure it correctly.

I think it's better than adding one more process for that

Reply:
I take that back I was wrongly reading the docs in
https://vector.dev/docs/reference/configuration/sinks/prometheus_exporter/

We can configure identical, and can reuse the same port as syslong exporter